setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name='libero schemas',
```

Comment:
```suggestion
    name='Libero schemas',
```

Reply:
This is what will be in the list of packages after being installed. So currently it looks like `libero-schemas` If I commit this change it will be `Libero-schemas`. I prefer the lower case name, personally.

Reply:
Ah ok, thought this was mean to be a human name. In that case just go for `libero-schemas` here?